### DISCUSSION TITLE
原作者不更新了，朋友们有什么的办法吗

### DISCUSSION THREAD

**Discussion by FionViber** [zetaloop/ExplorerPatcher#18] [General]
原作者自己说不更新了，现在能找到的替代品就是startallback，但是startallback太笨重了，搞得系统卡卡的，各位还有别的什么替代品吗？或者是有人接手了这个项目继续更新吗？

**Comment by czz404** [comment]
今天原作者更新了😊

**Reply by zetaloop** [reply]
感谢！
